functions-methods.js

```diff
@@ -52,17 +34,7 @@ console.log(emailIncludes)
 // checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
 
 
-function checkEmailValidity(email2) {
-    if (email2.includes('@')) {
-        return true;
-    } else if ((email2.includes(',')) && (email2.lastIndexOf('.'))) {
-        return false;
-    }
-}
-
-const outcomeValid = checkEmailValidity("n.eeken@novi.nl");
 
```

Comment:
Hiermee heb je nog niet afgevangen dat een emailadres dat een @ bevat, ook een , of een . op de laatste plaats.